ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -77,19 +77,49 @@ function getUrlParamsForEntity(selectEntity: {
 }
 
 export type NetworkGraphProps = {
-    detailType?: UrlDetailTypeValue;
-    detailId?: string;
     model: Model;
 };
 
 export type TopologyComponentProps = NetworkGraphProps;
 
-const TopologyComponent = ({ detailType, detailId, model }: TopologyComponentProps) => {
+function getNodeEdges(selectedNode) {
+    const egressEdges = selectedNode.getSourceEdges();
```

Comment:
The UI seems to break and it's being caused by this `getSourceEdges`. What is that function?

<img width="1552" alt="Screen Shot 2022-11-01 at 8 36 37 AM" src="https://user-images.githubusercontent.com/4805485/199273721-fa48a40c-31e4-4205-b8e4-8f85ac275646.png">